src/audio/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! Loading and playing of audio files.
+pub mod play;
+pub mod output;
+
+mod audio_context;
+mod source;
+
+pub use self::audio_context::AudioContext;
```

Comment:
I guess we don't need such a deep hierarchy. I would move source and context into this file, actually.